utils/src/main/kotlin/ProcessCapture.kt

```diff
@@ -35,16 +35,22 @@ class ProcessCapture(vararg command: String, workingDir: File? = null, environme
     constructor(workingDir: File?, vararg command: String) : this(*command, workingDir = workingDir)
 
     companion object {
-        private const val MAX_LOG_LINES = 20
-        private const val MAX_LOG_LINES_MESSAGE = "(Above output is limited to $MAX_LOG_LINES lines.)"
+        private const val MAX_OUTPUT_LINES = 20
+        private const val MAX_OUTPUT_FOOTER =
+                "(Above output is limited to $MAX_OUTPUT_LINES heading and tailing lines.)"
```

Comment:
Maybe say `to each $MAX_OUTPUT_LINES heading and tailing lines`.

Reply:
Done.